LICENSE.txt

```diff
@@ -12,16 +14,13 @@ Unless required by applicable law or agreed to in writing, software distributed
 ## Third Party Licenses ##
 
 ### Avalon Edit
-http://www.codeproject.com/Articles/42490/Using-AvalonEdit-WPF-Text-Editor
-http://opensource.org/licenses/lgpl-3.0.html
+http://avalonedit.net/
+https://opensource.org/licenses/lgpl-2.1.php
+  (LGPL-2.1 license)
```

Comment:
it looks like AvalonEdit is actually MIT:
https://github.com/icsharpcode/AvalonEdit/blob/master/LICENSE

Reply:
Lates version is, but the one we are using is LGPL-2.1. See https://www.nuget.org/packages/AvalonEdit/4.3.1.9430

Reply:
hmm, I wonder if it is worth it to check which version Revit ships these days and align if they have moved up to a more recent one.

Reply:
That's a good question. I guess currently we rely on them to test, but if it works we don't know if we could have upgraded. In that case we would be sticking with an old version for no reason.

Can we maybe run the D4R tests and see if causes problems? Or is it more complex than that?